IntroOOP/Builder/src/com/globant/builder/connection/Password.java

```diff
@@ -0,0 +1,15 @@
+package com.globant.builder.connection;
+
+import java.util.Scanner;
+
+public class Password extends ConnectionItem {
+  @Override
+  public String getValue() {
```

Comment:
I see this block of code repeated many times. Refactor the code to avoid this

Reply:
Ok. I will! 
Thanks.